src/equality.rs

```diff
@@ -26,23 +26,23 @@ impl FromStr for SerializedEq {
     type Err = String;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        if let Some((l, r)) = s.split_once("=>") {
+        if let Some((l, r)) = s.split_once("<=>") {
```

Comment:
have to do the `<=>` case before the `=>` case otherwise everything will be parsed as `=>`

Reply:
Hilarious